src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -8,6 +8,8 @@ Release History
 
 0.2.10
 ++++++
+* webapp: az webapp up is updated to create a P1V1 SKU app if os is Linux
+* webapp, functionapp: Fixing the error where even when zip deployment fails the error message shows, deployment is taking too long
```

Comment:
Would it be possible to rephrase this? If not, that's ok.

Reply:
the res_dict doesn't return any error messages that's why had to remove this

Reply:
the response from the request does not have any error / message key if the deployment fails?

Reply:
Oh, I didn't know, thanks for the clarification